formicarium/formicarium/Lidar.py

```diff
@@ -16,8 +19,9 @@ def __init__(self, collider:ICollider, range: float, xPos: float, yPos: float) -
         self.position = (xPos, yPos)
         self.white = Color(255, 255, 255)
         self.red = Color(255, 0, 0)
+        self.m2p = 3779.5275590551
```

Comment:
we should move variables like this to a seperate comfig file

Reply:
yeah should make a seperate ticket for this